search/types_skipper.cpp

```diff
@@ -44,6 +52,7 @@ void TypesSkipper::SkipEmptyNameTypes(feature::TypesHolder & types) const
     if (m_isPoi(type))
       return false;
 
+    ftype::TruncValue(type, 2);
```

Comment:
The point here is about an exact match, and the comment before was:
```
Test for exact type (man_made-tower-communication is not).
```
Had a request to search for cellular towers :)

Reply:
There are many other tower types that could be unnamed (minor) but worth to search for, e.g. bell towers, observation towers, fort towers, pagodas
See https://wiki.openstreetmap.org/wiki/Key:tower:type

Reply:
So probably its best to index all of them until we separate them into more subtypes.

Reply:
We have only one 3-arity type now - communication.

Reply:
Yes, this is what I mean.
There are many search cases now under a `man_made=tower`.

(Which should ideally have their own types like a cell tower.)